build-tools/pom.xml

```diff
@@ -21,7 +21,7 @@
     <groupId>software.amazon.awssdk</groupId>
     <artifactId>build-tools</artifactId>
     <name>AWS Java SDK :: Build Tools</name>
-    <version>2.0.0-preview-4</version>
+    <version>1.0</version>
```

Comment:
I tend to agree that we can make this version independently but we should follow the same scheme for all our modules right? It seems strange to me to have build-tools @ `1.0` when everything else is `2.0.0`

Reply:
I choose 1.0 so it's different enough to not be confused why it isn't updated.